mitmproxy/flow/export.py

```diff
@@ -29,7 +40,7 @@ def curl_command(flow):
     data += "'%s'" % full_url
 
     if flow.request.content:
-        data += " --data-binary '%s'" % flow.request.content
+        data += " --data-binary '%s'" % _native(flow.request.content)
```

Comment:
Because we're using that data as a part of a string? - that makes sense.

Does that mean, we don't even need `_native`? What will happen in [these](https://github.com/dufferzafar/mitmproxy/blob/py3-flow-export/mitmproxy/flow/export.py#L151-L155) [cases](https://github.com/dufferzafar/mitmproxy/blob/py3-flow-export/mitmproxy/flow/export.py#L99)?